test/data/zimfile&other.zim

```diff
@@ -0,0 +1 @@
+zimfile.zim
```

Comment:
Is this (an invalid new test ZIM file) intentional or a result of some mistake/error?

Reply:
This is a git link to zimfile.zim. It will be a symlink on linux and whatever kind of link on windows :)